index.js

```diff
@@ -119,10 +149,12 @@ export default class OneSignal {
         invariant(
             type === NOTIFICATION_RECEIVED_EVENT ||
             type === NOTIFICATION_OPENED_EVENT ||
-            type === IDS_AVAILABLE_EVENT ||
+//            type === IDS_AVAILABLE_EVENT ||
```

Comment:
can we delete this comment code? or is it something we want in the future? in that case, maybe add a comment?

Reply:
I think we need to keep `IDS_AVAILABLE_EVENT` here. It is the `'ids'` event correct?

Reply:
fixed this